tests/test_copies.py

```diff
@@ -0,0 +1,58 @@
+import pytest
+import os
+import shutil
+from shallow_backup import _copy_file, copy_dir
+from constants import Constants
+
+TEST_DIR = 'test-directory'
+TEST_BACKUP_DIR = 'test-backup-dir'
+TEST_TEXT_FILE = 'test-file.txt'
+DIRS = [TEST_DIR, TEST_BACKUP_DIR]
+
+
+class TestCopyMethods:
+    """
+    Test the functionality of copying
+    """
+
+    def setup_method(self):
+        for directory in DIRS:
+            try:
+                os.mkdir(directory)
+            except FileExistsError:
```

Comment:
If the directories exist already, instead of removing them immediately (since the teardown methods should remove them, so if they exist, it's possible the directories weren't created by us), we should print an error message and ask the user whether they want to remove the files or exit, using the `inquirer` library in the same style as the CLI.

Reply:
Although I'm open to talk about this one. With the file name change to be much less generic, the prompt may be overkill and could interfere with scripting. What do you think?

Reply:
Yeah I did this so that if it was ever run in a CI environment we shouldn't expect to have any issues. A prompt in a CI environment would hang the build.

We can try to be smarter about this and append the current datetime to definitely avoid conflicts and then remove all dirs with a suffix `shallow-copy-dir-*`

Reply:
TBH I think that might be a bit of over-engineering for a simple unit test solution. Going to keep as is for now but I'm happy to throw it is (simple change) if you deem necessary

Reply:
> We can try to be smarter about this and append the current datetime to definitely avoid conflicts and then remove all dirs with a suffix `shallow-copy-dir-*`

This is a great solution -- let's do it.

Reply:
Or I'm also comfortable going with `shallow-backup-test-dir` because it's very unlikely that someone has a pre-existing dir named that which isn't used by `shallow-backup`.